### PR TITLE
Add first version of the efficient decoder

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
- Reorganize the order of upscaling and conv layers so the conv layers operate on a smaller image and do equivalent work
- Look at layer wise profiling results from the TVM profiler on our vulkan deployment
- Reduce number of channels in the decoder and add a 1x1 conv which reduces the channels after the concatenation of the skip connections.
- Optimize the TVM compilation
- Use mixed precision

These changes bring the performance on the AMD Vega ATI 05:00.0 Lucienne (GPU of the Ryzen 7 5700U) with the Vulcan API from ~20 FPS to ~90 FPS. The accuracy in full precision is identical. Half needs to be evaluated.

### CODE REVIEW

File: config/yoeo_light_decoder.cfg
Comment (phinik) [inline config/yoeo_light_decoder.cfg:1]: Maybe use a more expressive name for the cfg file, or we could start to add a "comment" section to each cfg file explaining in what way it differs from the others.

File: yoeo/scripts/convertONNXModelToTVM.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    from tvm.driver import tvmc
+    from tvm.driver.tvmc.model import TVMCModel
+    from tvm.relay.transform import ToMixedPrecision
+    from tvm import relay
+except ModuleNotFoundError:
+    print("Please install Apache TVM!")
+    sys.exit(1)
+
+
+def make_parser():
```
Comment (phinik) [inline yoeo/scripts/convertONNXModelToTVM.py:17]: Maybe add another argument to disable tuning, for example if you just want to create a tvm model for debugging / testing of code

**Review by phinik — APPROVED** [review]

**Comment by jaagut** [comment]
What's the status of this? Is it ready for a merger?

**Comment by Flova** [comment]
I bumped some versions and the ci fails

**Pull request merged**